gapic/src/main/com/google/gapid/models/ConstantSets.java

```diff
@@ -94,6 +95,9 @@ public void onReplayDeviceChanged(Instance dev) {
     if (!Pods.mayBeConstant(value)) {
       return Service.Constant.getDefaultInstance();
     }
+    if (Objects.isNull(constants)) {
+      return Service.Constant.getDefaultInstance();
+    }
```

Comment:
Didn't reproduce it but it's good to have this null check in. Only a nit that `constants != null` is enough and it can be combined with the check above.